src/util/config.js

```diff
@@ -1,97 +1,168 @@
-import Joi from 'joi';
+import fs from 'fs';
```

Comment:
It wouldn't hurt to test any custom validation functions we write.
Otherwise yeah, the only other thing I would test is to confirm it returns a valid JS object.